CHANGELOG.md

```diff
@@ -5,6 +5,121 @@ information on the list of deprecated flags and APIs please have a look at
 https://docs.docker.com/engine/deprecated/ where target removal dates can also
 be found.
 
+## 17.07.0-ce (2017-07-20)
+
+### API & Client
+
+* Api: Update swagger.yaml for configs [moby/moby#34172](https://github.com/moby/moby/pull/34172)
+* Enable pprof/debug endpoints by default [moby/moby#32453](https://github.com/moby/moby/pull/32453)
+* Unexport subcommands [docker/cli#350](https://github.com/docker/cli/pull/350)
+- Fix filters displaying for docker search [docker/cli#331](https://github.com/docker/cli/pull/331)
+* Password stdin [docker/cli#271](https://github.com/docker/cli/pull/271)
+* Don't attempt to remove unsupported resources on older daemon [docker/cli#277](https://github.com/docker/cli/pull/277)
+* Return an empty stats if "container not found" [moby/moby#34004](https://github.com/moby/moby/pull/34004)
++ Add cluster events details in swagger.yml [moby/moby#34035](https://github.com/moby/moby/pull/34035)
+* Matching events using ID and name [moby/moby#33377](https://github.com/moby/moby/pull/33377)
+- Fix stack output stream [docker/cli#298](https://github.com/docker/cli/pull/298)
+* Progress: Show task error in place of progress bar [docker/cli#259](https://github.com/docker/cli/pull/259)
+* Login: print a big warning when using --password [docker/cli#270](https://github.com/docker/cli/pull/270)
++ Add `--detach` to docker scale [docker/cli#243](https://github.com/docker/cli/pull/243)
+* Set ping version even on error [moby/moby#33827](https://github.com/moby/moby/pull/33827)
+* Progress: Show success message when finished counting down [docker/cli#260](https://github.com/docker/cli/pull/260)
```

Comment:
Perhaps we should combine the "Progress" bullets to a single "improve presentation of progress bars in interactive mode"